### ISSUE TITLE
feedback ass2

### ISSUE THREAD

**Issue by gosiamigut** [vlpreda1/Machine-Learning#2]
Total  11 out of 15 = 73%
Written assignment: 3.5/5
+ see the solutions online
Programming assignment: 7.5/10
+ well done
+ implementation
  + for the default scores it would also be good to print the cv-scores to see if you are not overfitting
  + C is not a learning rate but the regularization parameter
  + the scores between default and optimization differ because your default is just on the full training set while the optimized is on the cv set. try to re-run the default on cv-set and the average error will then be different.
+ analysis
  +there is hardly any analysis
  + which classifier performs the best, what are your plots showing, what does the confusion matrix say?
